definitions/checks/foreman_tasks/not_paused.rb

```diff
@@ -9,12 +9,20 @@ class NotPaused < ForemanMaintain::Check
     end
 
     def run
-      paused_tasks_count = feature(:foreman_tasks).paused_tasks_count(ignored_tasks)
+      paused_tasks_count = feature(:foreman_tasks).paused_tasks_count()
```

Comment:
As per suggestion, you have removed argument `ignored_tasks` while getting paused tasks count. Is this `ignored_tasks` method still needed? Probably leftover?

Reply:
Yes, It will become dead code. Removing it.